sysz

```diff
@@ -93,8 +95,13 @@ _sysz_run() {
   eval "$@" || return $?
 }
 
+_sysz_use_sudo() {
+  # returns true if '--no-sudo' flag is unset and if the user is not root
+  [[ $NO_SUDO != true ]] && [ $EUID -ne 0 ]
+}
+
 _sysz_systemctl() {
-  if [[ $EUID -ne 0 && $1 = --system ]]; then
```

Comment:
Is there a possible way to augment the check so that will work for everybody? Like a polkit command? I'd prefer something more automatic, since this is supposed to be a a quick and easy tool that doesn't require configuration.

Reply:
There's 
```sh 
pkcheck --enable-internal-agent -u -p $$ -a org.freedesktop.systemd1.manage-units
```
which could work.
I need to check on a VM without polkit-managed systemd how it behaves..